### DISCUSSION TITLE
Domain stats tease and CTA

### DISCUSSION THREAD

**Discussion by ackernaut** [ParabolInc/parabol#5666] [Ideas]
Short of building out the full enterprise usage report as admin views (which can be scoped as a broader design exploration) we could tease folks in their admin view with domain stats and a CTA to upgrade.

## User job story

>When I learn that our company has many accounts and many teams, that we are over usage, and I want to continue using Parabol with my teams, I want access to easy to understand data that doesn’t expose the content of my meetings and a clear set of options for what’s required of me, so that I can initiate the purchase process with my company.

## Open questions
- How might we start small, sharing some small subset of the enterprise report in the admin view today?
- What might we learn in order to build out more of the domain stats in the admin views?
- How could we test what stats are safe to share across accounts?
- Do we require changes to pricing? Is it okay to say that one domain has too many teams, or should we say companies can only have 1 account?

## Concepts

Here’s a rough concept from the prototype:

![Domain stats CTA](https://user-images.githubusercontent.com/307286/142079053-a9475b3f-8a69-4b12-a23c-8a36b6a484f1.png)

We could hack this into the billing CTA to be scrappy:

### Before

![Billing upgrade CTA before](https://user-images.githubusercontent.com/307286/142079785-cd00ca7a-1be5-4d34-b529-35aade7c51f2.png)

### After

![Billing upgrade CTA after domain stats is turned on for org](https://user-images.githubusercontent.com/307286/142079806-6398574b-874c-4744-b7ea-448be583ec09.png)

## Ideas for acceptance criteria
- Parabol admins can cherry pick organizations and turn on this information panel using a mutation
- The panel shows up in the existing organization view (to be designed, probably replaces the existing billing CTA) with some anonymous stats and a CTA
- Folks can push to upgrade (prompting a conversation with sales) or contact for more info (prompting the sales rep to share the enterprise usage report)
- This is meant to be a small start to showing more company usage stats in the admin view (instead of manually generating reports)
- Drafted issues for the mutation and adding the conditional stats component in the existing UI
- Does this require a specific language change on the pricing page? That companies can have no more than 1 account? Or is the number of teams across the company enough to justify the prompt to conversion?

## Estimated efforts
- For UX/UI design concerns and refinement, 13 hours
- For deciding on the proper workflow with sales?
- For scoping and writing the mutation?
- For adding the stats panel to the existing organization view?

**Comment by avivapinchas** [comment]
I like the idea of testing this! It would help us understand a few things: 
- In general, does the way we present this info makes sense? Have we figured out a good way to explain 'what is an account'? 
- It would let us test some options for driving organic upgrades - we could try sending them here via emails, offering diff discounts or offers, and basically work though what resonates

My big Q here is: **How do folks get here?**

There's 2 problems, imo: 
- "Organizations" is a bit buried as a view - views to these pages make up a teeny .36% of all pageviews in the app. It's just not a spot people go to. So, having something here could help us test if that messaging is effective, but we'd need to think about what drives folks there. 
- I think we get into the confusion of 'account' vs 'organization' - if this banner is visible on a particular organization, does it apply just to that org? is it confusing why it's not visible maybe on other orgs? I guess if we do a good-enough job of cherry picking the orgs, this might not be an issue.

**Reply by ackernaut** [reply]
Regarding understanding what an account is and what the information is saying. I think we can do a much better job here. I’d consider what we’ve done so far a ‘drunken house drawing’. I’m a-okay spelling things out for folks if it’ll work.
>You are the Billing Leader* of 1 of 4 accounts* (we call a single account an ‘Organization’ 🤓 ) associated with the Verik.io email domain*. Across the Verik.io domain there are 4 active accounts*, 24 active teams*, and 128 active seats*. Across teams at the Verik.io domain, you have run 128 Retro meetings and 24 Sprint Poker meetings. The Verik.io domain is over the free limit* of usage. Contact us to talk about an Enterprise account, or find out more information

Every nuanced term could have a pop-up with a definition and a link to a support page that spells out all of our account terminology in sections with linkable headers.

Regarding how to find this, a few ideas:
1) We could try a banner on the dashboard with a direct link to the billing CTA ‘action required’
2) We could email them with a direct link to the billing CTA ‘repeating the same info, action required’
3) If we lock down any teams etc. we could point here with a dialog ‘action required’

For what’s it worth I don’t think I have _the_ answers/solutions here, just some ideas. The main idea I would stress is that this seems ripe as an opportunity of testing something small and learning quickly.

**Reply by ackernaut** [reply]
There may be a need to clarify further if a company commonly uses many TLD for employee emails. [See Bruce’s note on Slack](https://parabol.slack.com/archives/C08FL5UE7/p1637359484211200?thread_ts=1637184492.203500&cid=C08FL5UE7) 🔒 

We could share that too: we see X active accounts using TLD1, TLD2 (with the other stats; number of teams, active seats, meetings, etc.)
